bitbots_body_behavior/src/bitbots_body_behavior/body_behavior.py

```diff
@@ -63,7 +74,16 @@
     rospy.Subscriber("move_base/result", MoveBaseActionResult, D.blackboard.pathfinding.status_callback)
 
     rate = Rate(125)
+    counter = 0
+    path_to_ball_service_response = None
     while not rospy.is_shutdown():
         D.update()
         D.blackboard.team_data.publish_strategy()
+        D.blackboard.team_data.publish_time_to_ball()
+        counter = (counter + 1) % 125
+        D.blackboard.pathfinding.path_to_ball_check(path_to_ball_service_response)
+        if counter == 0 and D.blackboard.gamestate.is_game_state_equals(GameState.GAMESTATE_PLAYING):
```

Comment:
Why do we only publish it in playing? It's most likely needed in playing, but what's wrong with sending it in the other game states?

Reply:
I agree, this is preliminary since the sanity checks like "is the robot localized" and "does the robot have a good estimate of the ball position" are still missing